src/main/java/com/nashtech/model/Car.java

```diff
@@ -0,0 +1,52 @@
+package com.nashtech.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Version;
+
+/**
+ * Represents a reactive data model for sending
+ * details of the Car to the CosmosDB.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Car {
+
+    /**
+     * The card ID of the car.
+     */
+    private Integer cardId;
+
+    /**
+     * The brand of the car.
+     */
+    private String brand;
+
+    /**
+     * The model of the car.
+     */
+    @Version
```

Comment:
This a data coming from third party then what is the relevance of using this annotation.